### PR TITLE
Support multi masters in a single AZ

### PR DESCRIPTION
**Pull request by justinsb**
We primarily need to make sure we don't mount all the volumes on the
same instance.

This is not complete support for multiple masters (we still block it in `kops create`, but it is a start)

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/1536)

### CODE REVIEW

**Comment by kris-nova** [comment]
Just curious - why are we touching volume mounting code in this PR?

**Comment by kris-nova** [comment]
Just kidding - I should really read before I comment

File: protokube/pkg/protokube/volume_mounter.go [hunk 1]
```diff
@@ -193,20 +191,50 @@ func (k *VolumeMountController) attachMasterVolumes() ([]*Volume, error) {
...
+
+		glog.V(2).Infof("Trying to mount master volume: %q", v.ID)
+
+		err := k.provider.AttachVolume(v)
+		if err != nil {
+			// We are racing with other instances here; this can happen
+			glog.Warningf("Error attaching volume %q: %v", v.ID, err)
+		} else {
+			if v.LocalDevice == "" {
+				glog.Fatalf("AttachVolume did not set LocalDevice")
```
Comment (kris-nova) [inline protokube/pkg/protokube/volume_mounter.go]: Doesn't a fatal through a `SIGTERM` at the process? Is that what we want? Just thinking for potential API/service days down the road

**Comment by justinsb** [comment]
Thanks!

The fatal was there before (and remember this is protokube, not kops, so it won't be a service).  The code path should be entirely unreachable, so I think we really do want to panic - something has gone very badly wrong, and so we want to pull the eject handle as fast as possible before we mount the wrong volume.

**Pull request merged**